src/repositories/LikeRepository.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-useless-catch */
+import database from '../models';
+
+const { Like, Accommodation } = database;
+
+class LikeRepository {
+  /**
+   * Creates a new Like.
+   * @param {object} user_id
+   * @param {object} accommodation_id
+   * @returns {object} Like.
+   */
+  static async Like({ user_id, accommodation_id }) {
+    try {
+      return await Like.create({
```

Comment:
When you have a `return` you don't need the await anymore, this is redundant. Try removing the `await` or assign it to a variable and return the variable instead.